Script Analysis Examples/Dataspace Analysis/DataSpace_Default_Analysis.ipynb

```diff
@@ -30,7 +30,6 @@
     "import scrapbook as sb\n",
```

Comment:
@vignesh-ni could you please refine the PR description? The reviewers might not be familiar with the recent changes.

Reply:
Updated